packages/wonder-blocks-core/components/unique-id-provider.js

```diff
@@ -34,7 +35,7 @@ type Props = {|
      * a mock IIdentifierFactory for the initial render, and then a unique ID
      * factory thereafter.
      */
-    mockOnFirstRender?: boolean,
+    mockOnFirstRender: boolean,
```

Comment:
I believe this is a breaking change because clients using this component may not have set this in the past.  In practice though it won't break anything b/c we currently aren't using wonder-blocks' flow types yet in webapp, see https://phabricator.khanacademy.org/D49926 for details.